app/layout.tsx

```
@@ -1,5 +1,6 @@
 import type { Metadata } from "next";
 import { Geist, Geist_Mono } from "next/font/google";
+import { Toaster } from "@/components/ui/sonner"
 import "./globals.css";
 import { Toaster } from "@/components/ui/sonner"
```

Biome (2.1.2)

[error] Shouldn't redeclare 'Toaster'. Consider to delete it or rename it.

'Toaster' is defined here:


(lint/suspicious/noRedeclare)